tests/datasets/tasks/test_process_uploaded_file.py

```diff
@@ -49,6 +49,17 @@ def data(request):
     return request.param
 
 
+qualitative_data = [
+    ("GEOCODE_1", "F1_value_1"),
+    ("GEOCODE_2", "F1_value_2"),
+]
+qualitative_data_header = ["Geography", "content"]
+
+@pytest.fixture(params=[(qualitative_data, qualitative_data_header, "utf8")])
```

Comment:
Why do you do this instead of simply returning a tuple? Are you planning on additional parameters in future?